sdk/azidentity/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 ### Breaking Changes
+* Removed `AuthorizationCodeCredential`
```

Comment:
Maybe add a small note of what they should use instead - sounds like it should be InteractiveBrowserCredential?